proposals/subgroups.md

```diff
@@ -0,0 +1,227 @@
+# Subgroups
+
+Status: **Draft**
+
+Last modified: 2023-11-07
+
+Issue: #4306
+
+# Requirements
+
+**Vulkan**:
+* Vulkan 1.1, and
+* subgroupSupportedStages includes compute and fragment bit, and
+* subgroupSupportedOperations includes the following bits: basic, vote, ballot, shuffle, shuffle relative, arithmetic, quad, and
+* Vulkan 1.3 or VK_EXT_subgroup_size_control
+
+According to [this query](https://vulkan.gpuinfo.org/displaycoreproperty.php?core=1.1&name=subgroupSupportedOperations&platform=all),
+~84% of devices are captured.
```

Comment:
It looks like these percentages are fractions of unique reports.

It would be more meaningful to weight by shipped (and active) units.  But this data is hard to get.  (As a committee we should decide on a set of representative popular devices. But that goes way beyond this issue.)

I think generally the guidance works out ok though.